File: CHPT/source/trunk/supp_app/SspDispatchService/src/com/ctfo/storage/model/basedata/TbCustomer.java
package com.ctfo.storage.model.basedata;

import java.io.Serializable;
import java.math.BigDecimal;

/**
 * 
 * 
 * <p>
 * ----------------------------------------------------------------------------- <br>
 * 工程名 ： SspDispatchService <br>
 * 功能： 客户档案<br>
 * 描述： 客户档案<br>
 * 授权 : (C) Copyright (c) 2011 <br>
 * 公司 : 北京中交慧联信息科技有限公司 <br>
 * ----------------------------------------------------------------------------- <br>
 * 修改历史 <br>
 * <table width="432" border="1">
 * <tr>
 * <td>版本</td>
 * <td>时间</td>
 * <td>作者</td>
 * <td>改变</td>
 * </tr>
 * <tr>
 * <td>1.0</td>
 * <td>2014-11-5</td>
 * <td>xuehui</td>
 * <td>创建</td>
 * </tr>
 * </table>
 * <br>
 * <font color="#FF0000">注意: 本内容仅限于[北京中交慧联信息科技有限公司]内部使用，禁止转发</font> <br>
 * 
 * @version 1.0
 * 
 * @author xuehui
 * @since JDK1.6
 */
public class TbCustomer implements Serializable {

	/** */
	private static final long serialVersionUID = -2804759508090449642L;

	/** 客户ID */
	private String custId;

	/** 客户编码 */
	private String custCode;

	/** 客户名称 */
	private String custName;

	/** 客户简称 */
	private String custShortName;

	/** 简称首拼速查码 */
	private String custQuickCode;

	/** 客户类别，关联字典表关联 */
	private String custType;

	/** 法人 */
	private String legalPerson;

	/** 所属省，关联字典码表 */
	private String province;

	/** 所属市，关联字典码表 */
	private String city;

	/** 所属区县，关联字典码表 */
	private String county;

	/** 联系地址 */
	private String custAddress;

	/** 邮编 */
	private String zipCode;

	/** 固话 */
	private String custTel;

	/** 手机 */
	private String custPhone;

	/** 传真 */
	private String custFax;

	/** 电子邮箱 */
	private String custEmail;

	/** 网址 */
	private String custWebsite;

	/** 企业性质，关联码表字典 */
	private String enterpriseNature;

	/** 纳税人识别号 */
	private String taxNum;

	/** 信用等级，关联字码表典 */
	private String creditRating;

	/** 信用额度 */
	private Integer creditLine;

	/** 信用账期 */
	private Integer creditAccountPeriod;

	/** 价格类型，关联字典码表 */
	private String priceType;

	/** 开户银行，关联字典码表 */
	private String openBank;

	/** 银行账号 */
	private String bankAccount;

	/** 银行账号开户人 */
	private String bankAccountPerson;

	/** 开票名称 */
	private String billingName;

	/** 开票地址 */
	private String billingAddress;

	/** 开票账号 */
	private String billingAccount;

	/** 备注 */
	private String custRemark;

	/** 是否会员 */
	private String isMember;

	/** 会员编号 */
	private String memberNumber;

	/** 会员等级，关联字典码表 */
	private String memberClass;

	/** 会员有效期 */
	private Long memberPeriodValidity;

	/** 配件享受折扣 */
	private BigDecimal accessoriesDiscount;

	/** 工时费享受折扣 */
	private BigDecimal workhoursDiscount;

	/** 状态 0，停用 1，启用 */
	private String status;

	/** 删除标记，0删除，1未删除 默认1 */
	private String enableFlag;

	/** 宇通SAP代码 */
	private String ytSapCode;

	/** 宇通客户经理 */
	private String ytCustomerManager;

	/** 数据来源，关联字典码表 */
	private String dataSource;

	/** 国家---宇通 */
	private String country;

	/** 客户关系综述---宇通 */
	private String custRelation;

	/** 独立法人---宇通 */
	private String indepenLegalperson;

	/** 细分市场---宇通 */
	private String marketSegment;

	/** 组织机构代码（工商）---宇通 */
	private String institutionCode;

	/** 公司体制---宇通 */
	private String comConstitution;

	/** 注册资金(万)---宇通 */
	private String registeredCapital;

	/** 车辆结构---宇通 */
	private String vehicleStructure;

	/** 经销商---宇通 */
	private String agency;

	/** 客户sap代码---宇通 */
	private String sapCode;

	/** 业务经营范围---宇通 */
	private String businessScope;

	/** 企业资质---宇通 */
	private String entQualification;

	/** CRM客户ID */
	private String custCrmGuid;

	/** 创建人，关联人员表 */
	private String createBy;

	/** 创建时间 */
	private Long createTime;

	/** 最后编辑人，关联人员表 */
	private String updateBy;

	/** 最后编辑时间 */
	private Long updateTime;

	/** 服务站id，云平台用 */
	private String serStationId;

	/** 帐套id，云平台用 */
	private String setBookId;

	/** 车主电话---宇通 */
	// private String contPhone;

	public String getSerStationId() {
		return serStationId;
	}

	public void setSerStationId(String serStationId) {
		this.serStationId = serStationId;
	}

	public String getSetBookId() {
		return setBookId;
	}

	public void setSetBookId(String setBookId) {
		this.setBookId = setBookId;
	}

	public String getCustId() {
		return custId;
	}

	public void setCustId(String custId) {
		this.custId = custId;
	}

	public String getCustCode() {
		return custCode;
	}

	public void setCustCode(String custCode) {
		this.custCode = custCode;
	}

	public String getCustName() {
		return custName;
	}

	public void setCustName(String custName) {
		this.custName = custName;
	}

	public String getCustShortName() {
		return custShortName;
	}

	public void setCustShortName(String custShortName) {
		this.custShortName = custShortName;
	}

	public String getCustQuickCode() {
		return custQuickCode;
	}

	public void setCustQuickCode(String custQuickCode) {
		this.custQuickCode = custQuickCode;
	}

	public String getCustType() {
		return custType;
	}

	public void setCustType(String custType) {
		this.custType = custType;
	}

	public String getLegalPerson() {
		return legalPerson;
	}

	public void setLegalPerson(String legalPerson) {
		this.legalPerson = legalPerson;
	}

	public String getProvince() {
		return province;
	}

	public void setProvince(String province) {
		this.province = province;
	}

	public String getCity() {
		return city;
	}

	public void setCity(String city) {
		this.city = city;
	}

	public String getCounty() {
		return county;
	}

	public void setCounty(String county) {
		this.county = county;
	}

	public String getCustAddress() {
		return custAddress;
	}

	public void setCustAddress(String custAddress) {
		this.custAddress = custAddress;
	}

	public String getZipCode() {
		return zipCode;
	}

	public void setZipCode(String zipCode) {
		this.zipCode = zipCode;
	}

	public String getCustTel() {
		return custTel;
	}

	public void setCustTel(String custTel) {
		this.custTel = custTel;
	}

	public String getCustPhone() {
		return custPhone;
	}

	public void setCustPhone(String custPhone) {
		this.custPhone = custPhone;
	}

	public String getCustFax() {
		return custFax;
	}

	public void setCustFax(String custFax) {
		this.custFax = custFax;
	}

	public String getCustEmail() {
		return custEmail;
	}

	public void setCustEmail(String custEmail) {
		this.custEmail = custEmail;
	}

	public String getCustWebsite() {
		return custWebsite;
	}

	public void setCustWebsite(String custWebsite) {
		this.custWebsite = custWebsite;
	}

	public String getEnterpriseNature() {
		return enterpriseNature;
	}

	public void setEnterpriseNature(String enterpriseNature) {
		this.enterpriseNature = enterpriseNature;
	}

	public String getTaxNum() {
		return taxNum;
	}

	public void setTaxNum(String taxNum) {
		this.taxNum = taxNum;
	}

	public String getCreditRating() {
		return creditRating;
	}

	public void setCreditRating(String creditRating) {
		this.creditRating = creditRating;
	}

	public Integer getCreditLine() {
		return creditLine;
	}

	public void setCreditLine(Integer creditLine) {
		this.creditLine = creditLine;
	}

	public Integer getCreditAccountPeriod() {
		return creditAccountPeriod;
	}

	public void setCreditAccountPeriod(Integer creditAccountPeriod) {
		this.creditAccountPeriod = creditAccountPeriod;
	}

	public String getPriceType() {
		return priceType;
	}

	public void setPriceType(String priceType) {
		this.priceType = priceType;
	}

	public String getOpenBank() {
		return openBank;
	}

	public void setOpenBank(String openBank) {
		this.openBank = openBank;
	}

	public String getBankAccount() {
		return bankAccount;
	}

	public void setBankAccount(String bankAccount) {
		this.bankAccount = bankAccount;
	}

	public String getBankAccountPerson() {
		return bankAccountPerson;
	}

	public void setBankAccountPerson(String bankAccountPerson) {
		this.bankAccountPerson = bankAccountPerson;
	}

	public String getBillingName() {
		return billingName;
	}

	public void setBillingName(String billingName) {
		this.billingName = billingName;
	}

	public String getBillingAddress() {
		return billingAddress;
	}

	public void setBillingAddress(String billingAddress) {
		this.billingAddress = billingAddress;
	}

	public String getBillingAccount() {
		return billingAccount;
	}

	public void setBillingAccount(String billingAccount) {
		this.billingAccount = billingAccount;
	}

	public String getCustRemark() {
		return custRemark;
	}

	public void setCustRemark(String custRemark) {
		this.custRemark = custRemark;
	}

	public String getIsMember() {
		return isMember;
	}

	public void setIsMember(String isMember) {
		this.isMember = isMember;
	}

	public String getMemberNumber() {
		return memberNumber;
	}

	public void setMemberNumber(String memberNumber) {
		this.memberNumber = memberNumber;
	}

	public String getMemberClass() {
		return memberClass;
	}

	public void setMemberClass(String memberClass) {
		this.memberClass = memberClass;
	}

	public Long getMemberPeriodValidity() {
		return memberPeriodValidity;
	}

	public void setMemberPeriodValidity(Long memberPeriodValidity) {
		this.memberPeriodValidity = memberPeriodValidity;
	}

	public BigDecimal getAccessoriesDiscount() {
		return accessoriesDiscount;
	}

	public void setAccessoriesDiscount(BigDecimal accessoriesDiscount) {
		this.accessoriesDiscount = accessoriesDiscount;
	}

	public BigDecimal getWorkhoursDiscount() {
		return workhoursDiscount;
	}

	public void setWorkhoursDiscount(BigDecimal workhoursDiscount) {
		this.workhoursDiscount = workhoursDiscount;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public String getEnableFlag() {
		return enableFlag;
	}

	public void setEnableFlag(String enableFlag) {
		this.enableFlag = enableFlag;
	}

	public String getYtSapCode() {
		return ytSapCode;
	}

	public void setYtSapCode(String ytSapCode) {
		this.ytSapCode = ytSapCode;
	}

	public String getYtCustomerManager() {
		return ytCustomerManager;
	}

	public void setYtCustomerManager(String ytCustomerManager) {
		this.ytCustomerManager = ytCustomerManager;
	}

	public String getDataSource() {
		return dataSource;
	}

	public void setDataSource(String dataSource) {
		this.dataSource = dataSource;
	}

	public String getCountry() {
		return country;
	}

	public void setCountry(String country) {
		this.country = country;
	}

	public String getCustRelation() {
		return custRelation;
	}

	public void setCustRelation(String custRelation) {
		this.custRelation = custRelation;
	}

	public String getIndepenLegalperson() {
		return indepenLegalperson;
	}

	public void setIndepenLegalperson(String indepenLegalperson) {
		this.indepenLegalperson = indepenLegalperson;
	}

	public String getMarketSegment() {
		return marketSegment;
	}

	public void setMarketSegment(String marketSegment) {
		this.marketSegment = marketSegment;
	}

	public String getInstitutionCode() {
		return institutionCode;
	}

	public void setInstitutionCode(String institutionCode) {
		this.institutionCode = institutionCode;
	}

	public String getComConstitution() {
		return comConstitution;
	}

	public void setComConstitution(String comConstitution) {
		this.comConstitution = comConstitution;
	}

	public String getRegisteredCapital() {
		return registeredCapital;
	}

	public void setRegisteredCapital(String registeredCapital) {
		this.registeredCapital = registeredCapital;
	}

	public String getVehicleStructure() {
		return vehicleStructure;
	}

	public void setVehicleStructure(String vehicleStructure) {
		this.vehicleStructure = vehicleStructure;
	}

	public String getAgency() {
		return agency;
	}

	public void setAgency(String agency) {
		this.agency = agency;
	}

	public String getSapCode() {
		return sapCode;
	}

	public void setSapCode(String sapCode) {
		this.sapCode = sapCode;
	}

	public String getBusinessScope() {
		return businessScope;
	}

	public void setBusinessScope(String businessScope) {
		this.businessScope = businessScope;
	}

	public String getEntQualification() {
		return entQualification;
	}

	public void setEntQualification(String entQualification) {
		this.entQualification = entQualification;
	}

	public String getCustCrmGuid() {
		return custCrmGuid;
	}

	public void setCustCrmGuid(String custCrmGuid) {
		this.custCrmGuid = custCrmGuid;
	}

	public String getCreateBy() {
		return createBy;
	}

	public void setCreateBy(String createBy) {
		this.createBy = createBy;
	}

	public Long getCreateTime() {
		return createTime;
	}

	public void setCreateTime(Long createTime) {
		this.createTime = createTime;
	}

	public String getUpdateBy() {
		return updateBy;
	}

	public void setUpdateBy(String updateBy) {
		this.updateBy = updateBy;
	}

	public Long getUpdateTime() {
		return updateTime;
	}

	public void setUpdateTime(Long updateTime) {
		this.updateTime = updateTime;
	}

}

File: CHPT/source/tags/V0.4.1/client_app/Model/tb_customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Model
{
    //客户档案
    public class tb_customer
    {
        /// <summary>
        /// 客户ID
        /// </summary>		
        private string _cust_id;
        public string cust_id
        {
            get { return _cust_id; }
            set { _cust_id = value; }
        }
        /// <summary>
        /// 客户编码
        /// </summary>		
        private string _cust_code;
        public string cust_code
        {
            get { return _cust_code; }
            set { _cust_code = value; }
        }
        /// <summary>
        /// 客户名称
        /// </summary>		
        private string _cust_name;
        public string cust_name
        {
            get { return _cust_name; }
            set { _cust_name = value; }
        }
        /// <summary>
        /// 客户简称
        /// </summary>		
        private string _cust_short_name;
        public string cust_short_name
        {
            get { return _cust_short_name; }
            set { _cust_short_name = value; }
        }
        /// <summary>
        /// 简称首拼速查码
        /// </summary>		
        private string _cust_quick_code;
        public string cust_quick_code
        {
            get { return _cust_quick_code; }
            set { _cust_quick_code = value; }
        }
        /// <summary>
        /// 客户类别，关联字典表关联
        /// </summary>		
        private string _cust_type;
        public string cust_type
        {
            get { return _cust_type; }
            set { _cust_type = value; }
        }
        /// <summary>
        /// 法人
        /// </summary>		
        private string _legal_person;
        public string legal_person
        {
            get { return _legal_person; }
            set { _legal_person = value; }
        }
        /// <summary>
        /// 所属省，关联字典码表
        /// </summary>		
        private string _province;
        public string province
        {
            get { return _province; }
            set { _province = value; }
        }
        /// <summary>
        /// 所属市，关联字典码表
        /// </summary>		
        private string _city;
        public string city
        {
            get { return _city; }
            set { _city = value; }
        }
        /// <summary>
        /// 所属区县，关联字典码表
        /// </summary>		
        private string _county;
        public string county
        {
            get { return _county; }
            set { _county = value; }
        }
        /// <summary>
        /// 联系地址
        /// </summary>		
        private string _cust_address;
        public string cust_address
        {
            get { return _cust_address; }
            set { _cust_address = value; }
        }
        /// <summary>
        /// 邮编
        /// </summary>		
        private string _zip_code;
        public string zip_code
        {
            get { return _zip_code; }
            set { _zip_code = value; }
        }
        /// <summary>
        /// 固话
        /// </summary>		
        private string _cust_tel;
        public string cust_tel
        {
            get { return _cust_tel; }
            set { _cust_tel = value; }
        }
        /// <summary>
        /// 手机
        /// </summary>		
        private string _cust_phone;
        public string cust_phone
        {
            get { return _cust_phone; }
            set { _cust_phone = value; }
        }
        /// <summary>
        /// 传真
        /// </summary>		
        private string _cust_fax;
        public string cust_fax
        {
            get { return _cust_fax; }
            set { _cust_fax = value; }
        }
        /// <summary>
        /// 电子邮箱
        /// </summary>		
        private string _cust_email;
        public string cust_email
        {
            get { return _cust_email; }
            set { _cust_email = value; }
        }
        /// <summary>
        /// 网址
        /// </summary>		
        private string _cust_website;
        public string cust_website
        {
            get { return _cust_website; }
            set { _cust_website = value; }
        }
        /// <summary>
        /// 企业性质，关联码表字典
        /// </summary>		
        private string _enterprise_nature;
        public string enterprise_nature
        {
            get { return _enterprise_nature; }
            set { _enterprise_nature = value; }
        }
        /// <summary>
        /// 纳税人识别号
        /// </summary>		
        private string _tax_num;
        public string tax_num
        {
            get { return _tax_num; }
            set { _tax_num = value; }
        }
        /// <summary>
        /// 信用等级，关联字码表典
        /// </summary>		
        private string _credit_rating;
        public string credit_rating
        {
            get { return _credit_rating; }
            set { _credit_rating = value; }
        }
        /// <summary>
        /// 信用额度
        /// </summary>		
        private int _credit_line;
        public int credit_line
        {
            get { return _credit_line; }
            set { _credit_line = value; }
        }
        /// <summary>
        /// 信用账期
        /// </summary>		
        private int _credit_account_period;
        public int credit_account_period
        {
            get { return _credit_account_period; }
            set { _credit_account_period = value; }
        }
        /// <summary>
        /// 价格类型，关联字典码表
        /// </summary>		
        private string _price_type;
        public string price_type
        {
            get { return _price_type; }
            set { _price_type = value; }
        }
        /// <summary>
        /// 开户银行，关联字典码表
        /// </summary>		
        private string _open_bank;
        public string open_bank
        {
            get { return _open_bank; }
            set { _open_bank = value; }
        }
        /// <summary>
        /// 银行账号
        /// </summary>		
        private string _bank_account;
        public string bank_account
        {
            get { return _bank_account; }
            set { _bank_account = value; }
        }
        /// <summary>
        /// 银行账号开户人
        /// </summary>		
        private string _bank_account_person;
        public string bank_account_person
        {
            get { return _bank_account_person; }
            set { _bank_account_person = value; }
        }
        /// <summary>
        /// 开票名称
        /// </summary>		
        private string _billing_name;
        public string billing_name
        {
            get { return _billing_name; }
            set { _billing_name = value; }
        }
        /// <summary>
        /// 开票地址
        /// </summary>		
        private string _billing_address;
        public string billing_address
        {
            get { return _billing_address; }
            set { _billing_address = value; }
        }
        /// <summary>
        /// 开票账号
        /// </summary>		
        private string _billing_account;
        public string billing_account
        {
            get { return _billing_account; }
            set { _billing_account = value; }
        }
        /// <summary>
        /// 备注
        /// </summary>		
        private string _cust_remark;
        public string cust_remark
        {
            get { return _cust_remark; }
            set { _cust_remark = value; }
        }
        /// <summary>
        /// 是否会员
        /// </summary>		
        private string _is_member;
        public string is_member
        {
            get { return _is_member; }
            set { _is_member = value; }
        }
        /// <summary>
        /// 会员编号
        /// </summary>		
        private string _member_number;
        public string member_number
        {
            get { return _member_number; }
            set { _member_number = value; }
        }
        /// <summary>
        /// 会员等级，关联字典码表
        /// </summary>		
        private string _member_class;
        public string member_class
        {
            get { return _member_class; }
            set { _member_class = value; }
        }
        /// <summary>
        /// 会员有效期
        /// </summary>		
        private long _member_period_validity;
        public long member_period_validity
        {
            get { return _member_period_validity; }
            set { _member_period_validity = value; }
        }
        /// <summary>
        /// 配件享受折扣
        /// </summary>		
        private decimal _accessories_discount;
        public decimal accessories_discount
        {
            get { return _accessories_discount; }
            set { _accessories_discount = value; }
        }
        /// <summary>
        /// 工时费享受折扣
        /// </summary>		
        private decimal _workhours_discount;
        public decimal workhours_discount
        {
            get { return _workhours_discount; }
            set { _workhours_discount = value; }
        }
        /// <summary>
        /// 状态   0，停用   1，启用
        /// </summary>		
        private string _status;
        public string status
        {
            get { return _status; }
            set { _status = value; }
        }
        /// <summary>
        /// 删除标记，0删除，1未删除  默认1
        /// </summary>		
        private string _enable_flag;
        public string enable_flag
        {
            get { return _enable_flag; }
            set { _enable_flag = value; }
        }
        /// <summary>
        /// 宇通SAP代码
        /// </summary>		
        private string _yt_sap_code;
        public string yt_sap_code
        {
            get { return _yt_sap_code; }
            set { _yt_sap_code = value; }
        }
        /// <summary>
        /// 宇通客户经理
        /// </summary>		
        private string _yt_customer_manager;
        public string yt_customer_manager
        {
            get { return _yt_customer_manager; }
            set { _yt_customer_manager = value; }
        }
        /// <summary>
        /// 数据来源，关联字典码表
        /// </summary>		
        private string _data_source;
        public string data_source
        {
            get { return _data_source; }
            set { _data_source = value; }
        }
        /// <summary>
        /// 国家---宇通
        /// </summary>		
        private string _country;
        public string country
        {
            get { return _country; }
            set { _country = value; }
        }
        /// <summary>
        /// 客户关系综述---宇通
        /// </summary>		
        private string _cust_relation;
        public string cust_relation
        {
            get { return _cust_relation; }
            set { _cust_relation = value; }
        }
        /// <summary>
        /// 独立法人---宇通
        /// </summary>		
        private string _indepen_legalperson;
        public string indepen_legalperson
        {
            get { return _indepen_legalperson; }
            set { _indepen_legalperson = value; }
        }
        /// <summary>
        /// 细分市场---宇通
        /// </summary>		
        private string _market_segment;
        public string market_segment
        {
            get { return _market_segment; }
            set { _market_segment = value; }
        }
        /// <summary>
        /// 组织机构代码（工商）---宇通
        /// </summary>		
        private string _institution_code;
        public string institution_code
        {
            get { return _institution_code; }
            set { _institution_code = value; }
        }
        /// <summary>
        /// 公司体制---宇通
        /// </summary>		
        private string _com_constitution;
        public string com_constitution
        {
            get { return _com_constitution; }
            set { _com_constitution = value; }
        }
        /// <summary>
        /// 注册资金(万)---宇通
        /// </summary>		
        private string _registered_capital;
        public string registered_capital
        {
            get { return _registered_capital; }
            set { _registered_capital = value; }
        }
        /// <summary>
        /// 车辆结构---宇通
        /// </summary>		
        private string _vehicle_structure;
        public string vehicle_structure
        {
            get { return _vehicle_structure; }
            set { _vehicle_structure = value; }
        }
        /// <summary>
        /// 经销商---宇通
        /// </summary>		
        private string _agency;
        public string agency
        {
            get { return _agency; }
            set { _agency = value; }
        }
        /// <summary>
        /// 客户sap代码---宇通
        /// </summary>		
        private string _sap_code;
        public string sap_code
        {
            get { return _sap_code; }
            set { _sap_code = value; }
        }
        /// <summary>
        /// 业务经营范围---宇通
        /// </summary>		
        private string _business_scope;
        public string business_scope
        {
            get { return _business_scope; }
            set { _business_scope = value; }
        }
        /// <summary>
        /// 企业资质---宇通
        /// </summary>		
        private string _ent_qualification;
        public string ent_qualification
        {
            get { return _ent_qualification; }
            set { _ent_qualification = value; }
        }
        /// <summary>
        /// CRM客户ID
        /// </summary>		
        private string _cust_crm_guid;
        public string cust_crm_guid
        {
            get { return _cust_crm_guid; }
            set { _cust_crm_guid = value; }
        }
        /// <summary>
        /// 创建人，关联人员表
        /// </summary>		
        private string _create_by;
        public string create_by
        {
            get { return _create_by; }
            set { _create_by = value; }
        }
        /// <summary>
        /// 创建时间
        /// </summary>		
        private long _create_time;
        public long create_time
        {
            get { return _create_time; }
            set { _create_time = value; }
        }
        /// <summary>
        /// 最后编辑人，关联人员表
        /// </summary>		
        private string _update_by;
        public string update_by
        {
            get { return _update_by; }
            set { _update_by = value; }
        }
        /// <summary>
        /// 最后编辑时间
        /// </summary>		
        private long _update_time;
        public long update_time
        {
            get { return _update_time; }
            set { _update_time = value; }
        }

    }
}